inc/preprocess.inc

```diff
@@ -570,3 +571,25 @@ function nicsdru_nidirect_theme_preprocess_responsive_image_formatter(array &$va
     $variables['responsive_image']['#attributes']['caption'] = $caption;
   }
 }
+
+/**
+ * Implements template_preprocess_container().
+ */
+function nicsdru_nidirect_theme_preprocess_container(&$variables) {
+  /**
+   * Fix for HTML validation error caused by duplicate "edit-actions" ID that is present in the header search form and
+   * in the page feedback form. The container for submit buttons in both forms has the id "edit-actions". The duplicate
+   * IDs may be caused by an issue in core.
+   * 
+   * @see https://www.drupal.org/project/drupal/issues/1852090
+   */
+  if (isset($variables['element']['#id'])) {
+    $id = $variables['element']['#id'];
+    if (in_array($id, ['edit-actions'])) {
+      $id .= '--' . Crypt::randomBytesBase64(8);
```

Comment:
Try this instead? https://api.drupal.org/api/drupal/core!lib!Drupal!Component!Utility!Html.php/function/Html%3A%3AgetUniqueId/8.2.x

This is the D8 version of `drupal_html_id()` which was used in D7 to guarantee unique ids in a given render array even if some items shared the same id/key.